rotkehlchen/db/upgrades/v34_v35.py

```diff
@@ -108,6 +118,28 @@ def _add_blockchain_column_web3_nodes(cursor: 'DBCursor') -> None:
     cursor.execute('DROP TABLE web3_nodes_old')
 
 
+def _update_ignored_assets_identifiers_to_caip_format(cursor: 'DBCursor') -> None:
+    cursor.execute('SELECT value FROM multisettings WHERE name="ignored_asset";')
+    old_ids_to_caip_ids_mappings: List[Tuple[str, str]] = []
+    for (old_identifier,) in cursor:
+        if old_identifier is not None and old_identifier.startswith(ETHEREUM_DIRECTIVE):
```

Comment:
Can it ever be `None`? Hm ... the schema allows it so it's good to have it but now I wonder. Why do we allow it in the schema? Why not have a `NOT NULL` there?